src/Backend/Command.ts

```diff
@@ -41,4 +41,26 @@ class Command extends Array<string> {
   }
 };
 
-export {Command};
+/**
+ * Class representing `$ pipingCmd | pipedCmd` type of piped command chain.
```

Comment:
(optional) if you intended doxygen format,
```suggestion
 * @brief Class representing `$ pipingCmd | pipedCmd` type of piped command chain.
```

Reply:
Thanks. I will change that later in other PR!